### DISCUSSION TITLE
Embedding documentation content in another site?

### DISCUSSION THREAD

**Discussion by bakerfugu** [facebook/docusaurus#3908] [Q&A] [answered]
How can I embed content from my documentation in another site? I've tried searching for this but have only been able to find results regarding embedding other content within the documentation, which I'm familiar with.

I have a code editor on a webpage and I would like to be able to have a resizable/collapsible pane to the side of the editor that displays my docusaurus documentation. [Readthedocs seems to have a version of this](https://docs.readthedocs.io/en/stable/guides/embedding-content.html) feature, and I'm curious if Docusarus has an equivalent or a workaround. 

Based on the wording "Every page component should export a React component" from [this part](https://v2.docusaurus.io/docs/creating-pages#using-react) of the docs, it seems like I can grab a React component for any page of my documentation, but I'm not sure how to do that / if that's the right approach.

**Comment by bakerfugu** [comment, marked as answer]
Alright, after spending more time on this it looks like this isn't possible. I misunderstood what "every page component should export a React component" meant.

It'd be nice if this was a thing though, along with the ability to use Docusaurus alongside an existing site.

**Reply by slorber** [reply]
Sorry but I have a hard time understand what you are looking for, please show me a screenshot.

You can embed any Docusaurus page in another website using an iframe, isn't this good enough for your usecase?

**Reply by dionjwa** [reply]
I think the goal is that when a docusaurus page is embedded, it just show the content, without the header, footer, sidebar, etc, otherwise it's a very confusing experience. Here is what docusaurus looks like embedded, it doesn't work at all as an embed:
![image](https://github.com/user-attachments/assets/4dbf344a-6f85-4286-bd4d-92c483f53616)

**Reply by slorber** [reply]
We have a feature to let embed pages in a better way:

https://docusaurus.io/docs/next/styling-layout#data-attributes

![CleanShot 2025-02-21 at 11 55 58](https://github.com/user-attachments/assets/4dd6b59f-2939-4d3b-838f-147cfa95c214)
